### ISSUE TITLE
No class name completion

### ISSUE THREAD

**Issue by sserbin** [lvht/phpcd.vim#108]
Test case:
Attempt completion on e.g. `RuntimeException` class:
```php
throw new RuntimeE<tab>
```
_Expected:_ `RuntimeException` class is completed
_Actual:_ you get a list of *all* defined constants and functions

This seems to be because the `PHPCD::info()` method does not handle class names completion at all - from the method docblock:
```
     * If only $pattern is setted, it will list all the defined function
     * (including the PHP's builtin function', filted by pattern.
```
It seems like it should list not only defined functions/constants but class names as well (or is there another method I've overlooked that should be handling class completion?).

Thanks

**Comment by lvht** [comment]
PR are welcome 😄

**Comment by lvht** [comment]
fixed by 9fa82c017c8347959c2432eb63608d9cee8febbf. However, you need type `throw new \Runtim<tab>`
